builtin/credential/chef-node/backend.go

```diff
@@ -0,0 +1,44 @@
+package chefNode
```

Comment:
Can we name this backend, just `chef`? This might increase the scope of the backend and will serve as placeholder for enhancements that are out of scope of this PR.